sdk/keyvault/keyvault-certificates/test/public/CRUD.spec.ts

```diff
@@ -98,22 +98,14 @@ describe("Certificates client - create, read, update and delete", () => {
 
   it("cannot create a certificate with an empty name", async function () {
     const certificateName = "";
-    let error;
     try {
       await client.beginCreateCertificate(
         certificateName,
         basicCertificatePolicy,
         testPollerProperties
       );
-      throw Error("Expecting an error but not catching one.");
-    } catch (e: any) {
-      error = e;
-    }
-    assert.equal(
-      error.message,
-      `"certificateName" with value "" should satisfy the constraint "Pattern": /^[0-9a-zA-Z-]+$/.`,
-      "Unexpected error while running beginCreateCertificate with an empty string as the name."
-    );
+      assert.fail("Expected an error");
+    } catch (e: any) {}
```

Comment:
nit: since we are not using `e` anymore, it should be fine to leave it as `unknown`?
```suggestion
    } catch (e) {}
```